pkg/gateway/meta/handler/grpc/handler.go

```diff
@@ -569,9 +582,20 @@ func (s *server) Insert(ctx context.Context, req *payload.Insert_Request) (loc *
 	}
 	uuid := fuid.String()
 	req.Vector.Id = uuid
+	if req.GetConfig().GetTimestamp() == 0 {
+		now := time.Now().UnixNano()
+		if req.GetConfig() == nil {
```

Comment:
I'm not sure but is that ok to check `req.GetConfig() == nil` after `req.GetConfig().GetTimestamp() == 0`?
Does nil pointer occur when `req.GetCnfig()` return nil?:thinking:

Reply:
it's ok.

Reply:
I got it. thank you.